release/ray_release/tests/test_global_config.py

```diff
@@ -19,7 +19,11 @@
   aws_cr: 029272617770.dkr.ecr.us-west-2.amazonaws.com
   gcp_cr: us-west1-docker.pkg.dev/anyscale-oss-ci
 state_machine:
-  aws_bucket: ray-ci-results
+  pr:
+    aws_bucket: ray-ci-pr-results
+  branch:
+    aws_bucket: ray-ci-results
+  disabled: 1
```

Comment:
why is this `aws_bucket` related change here?

Reply:
the previous schema definition was deprecated (the schema reader still supports both versions)